src/support/assertRender.ts

```diff
@@ -40,9 +53,18 @@ const diffOptions: DiffOptions = {
  *
  * @param actual The actual rendered DNode to be asserted
  * @param expected The expected DNode to be asserted against the actual
+ * @param options A set of options that effect the behaviour of `assertRender`
  * @param message Any message to be part of an error thrown if actual and expected do not match
  */
-export default function assertRender(actual: DNode, expected: DNode, message?: string): void {
+export default function assertRender(actual: DNode, expected: DNode, message?: string): void;
+export default function assertRender(actual: DNode, expected: DNode, options: AssertRenderOptions, message?: string): void;
+export default function assertRender(actual: DNode, expected: DNode, options: any, message?: string): void {
```

Comment:
Actually needs to be `options?: AssertRenderOptions | string` or `options: AssertRenderOptions | string | undefined`... I ran into an issue with the overloads and took the chicken out instead of really thinking about why it was complaining.